pkg/cloudprovider/providers/azure/azure_loadbalancer.go

```diff
@@ -948,6 +894,86 @@ func (az *Cloud) reconcileLoadBalancer(clusterName string, service *v1.Service,
 	return lb, nil
 }
 
+func (az *Cloud) createLoadBalancerRule(
```

Comment:
it's not actually create a rule, could we rename it to `reconcileLoadBalancerRule`?

Reply:
fixed